.gitlab-ci.yml

```diff
@@ -21,13 +21,17 @@ cache:
 
 .releaseable_branches:             # list of git refs for building GitLab artifacts (think "pre-release binaries")
   only:                            &releaseable_branches
-    - stable
-    - beta
-    - tags
+    refs:
+      - stable
+      - beta
+      - tags
+    variables:
+      - $SCHEDULE_TAG == "nightly"
```

Comment:
does this just return true if ref is nightly?

Reply:
It returns true if the variable $SCHEDULE_TAG is set to nightly, which will only be the case if this variable is explicitly set either in scheduleded builds or in manually triggered bulids.
For $CI_COMMIT_REF_NAME the behaviour will remain the same as before when pushing a nightly tag (but not needed anymore) - otherwise this variable will be the name of the branch which for scheduled nightly builds will be master.